scilpy/tracking/tracker.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, propagator: AbstractPropagator, mask: DataVolume,
             Whether to save the seeds associated to their respective
             streamlines.
         mmap_mode: str
-            Memory-mapping mode. One of {None, ‘r+’, ‘r’, ‘w+’, ‘c’}
+            Memory-mapping mode to access the tracker. One of {None, ‘r+’, ‘c’}
```

Comment:
It is for accessing the tracking data (for example SH coefficients image when tracking on ODF). Something like this?
```suggestion
            Memory-mapping mode. One of {None, 'r+', 'c'}. This value is passed to
            np.load() when loading the raw tracking data from a subprocess.
```